benchmark/runbenchmarks.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+julia -e 'import Pkg; Pkg.activate(get(ENV, "ONSAS", @__DIR__)); using PkgBenchmark; results = benchmarkpkg("ONSAS", retune=true); export_markdown("results.md", results)'
```

Comment:
```suggestion
julia -e 'import Pkg; Pkg.activate(get(ENV, "ONSAS", @__DIR__)); using PkgBenchmark; results = benchmarkpkg("ONSAS", retune=true); export_markdown("benchmark/results.md", results)'
```

Reply:
i wanted to avoid assuming that you are running the benchmark from within the ONSAS folder, but it's fine.